scripts/generate-docs-checkout.ts

```diff
@@ -17,10 +17,13 @@ const checkout = {
   shopifyDevAssets: '../shopify-dev/app/assets/images/api/checkout-extensions',
 };
 
-const componentsPageContent = (url: string, title = 'Checkout'): Content => ({
+const componentsPageContent = (
+  url: string,
+  title = 'checkout UI',
```

Comment:
Yeah, titles use sentence case in our docs, and [checkout should be lowercase ](https://shopify.slack.com/archives/C039DD1F9C3/p1652895964713549) otherwise